README.md

```diff
@@ -40,8 +40,12 @@ Many more companies are using Tart in their internal setups. Here are a few of t
   <a href="https://transloadit.com/" target=_blank>
     <img src="https://github.com/cirruslabs/tart/raw/main/Resources/Users/Transloadit.png" height="65"/>
   </a>
+  <a href="https://www.pitsdatarecovery.net/" target=_blank>
+    <img src="https://github.com/cirruslabs/tart/assets/77569421/b5a99e54-c59e-48c8-b6ef-411eda894a48" height="65"/>
```

Comment:
Interesting idea to use assets! Could you please check in the image though? Just for the reason of consistency with the rest of the assets in [`Resources/Users/` folder](https://github.com/cirruslabs/tart/tree/main/Resources/Users).